package.json

```diff
@@ -4,7 +4,7 @@
   "description": "The Trusona Forgerock Module is an Authentication Module for ForgeRock AM that allows you to authenticate users with Trusona.",
   "dependencies": {
     "@trusona/deeplink": "1.3.1",
-    "@trusona/trucode-svg": "~0.8",
+    "trusona-trucode": "~0.1.0",
```

Comment:
For future NPM usage, we should probably switch to using `^` instead of `~`, which if we follow semver should grab the latest compatible version.